plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -138,6 +138,7 @@ object ScalafmtPlugin extends AutoPlugin {
   private class FormatSession(
       config: Path,
       taskStreams: TaskStreams,
+      cacheStoreFactory: CacheStoreFactory,
```

Comment:
Not directly related to the fix, but I wonder if it would make sense to capture `sources` into this as well, and get rid of the `getScalafmtSourcesTask` wrapper? I can try in a follow-up PR.

Reply:
It's getting in the way of `scalafmtSbt` since we run on 2 different set of files within the same session, so I won't proceed with a PR